components/brave_wallet_ui/utils/network-utils.ts

```diff
@@ -56,7 +56,7 @@ type TxDataPresence = {
   ethTxData?: Partial<BraveWallet.TxDataUnion['ethTxData']> | undefined
   ethTxData1559?: Partial<BraveWallet.TxDataUnion['ethTxData1559']> | undefined
   solanaTxData?:
-    | Partial<BraveWallet.TxDataUnion['solanaTxData']>
+                 | Partial<BraveWallet.TxDataUnion['solanaTxData']>
```

Comment:
Ditto: Any idea what is causing this indentation?

Reply:
Fixed by moving to the same line as the property.